### PR TITLE
fix: only prints env key if there are env values to be passed.

### PR DESCRIPTION
**Pull request by jcchavezs**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

/kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

> If this PR will release a new chart version please make sure to also uncomment the following line:

/kind chart-release

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area falco-chart

> /area falco-exporter-chart

> /area falcosidekick-chart

> /area falco-talon

> /area event-generator-chart

> /area k8s-metacollector

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Outputting the env key without values is highlighted as error in vscode.

image

Fixes #

**Special notes for your reviewer**:

**Checklist**

- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] CHANGELOG.md updated

### CODE REVIEW

**Comment by alacuku** [comment]
Hi @jcchavezs, thanks for the contribution. Please follow the steps described in the `Contributing` section: https://github.com/falcosecurity/charts#contributing?

**Comment by jcchavezs** [comment]
Done @alacuku

File: charts/falco/CHANGELOG.md
Comment (alacuku) [inline charts/falco/CHANGELOG.md:1]: Could you please revert the formatting changes?

Comment (jcchavezs) [reply charts/falco/CHANGELOG.md:1]: sorry about that. Done.

**Review by alacuku — APPROVED** [review]
/lgtm

**Pull request merged**